packages/clerk-js/webpack.config.js

```diff
@@ -155,6 +155,9 @@ const commonForProd = () => {
       globalObject: 'globalThis',
     },
     plugins: [
+      new webpack.DefinePlugin({
+        __IS_BROWSER__: variant !== variants.clerkHeadless,
+      }),
```

Comment:
My understanding is that this is only used in `clerkHeadless` package, so let's kill the `variant` argument from here and move the plugin definition to the clerkHeadless rules in `prodConfig` (probably line 196?)